### ISSUE TITLE
Problem binding TIMESTAMP

### ISSUE THREAD

**Issue by ptamburro** [olavloite/spanner-jdbc#105]
Problem using a CURRENT_TIMESTAMP() spanner function.

Example.
INSERT INTO providers
(provider_id, created, device_id, document_number, document_type_id, name, notification, phone, updated, last_access, deleted, token_fcm)
VALUES('000000', CURRENT_DATE(), 'not-device-id', null, null, 'provider-promocion', false, null, CURRENT_TIMESTAMP(), CURRENT_TIMESTAMP(), false, 'not-token');

this query produces a SQL Error: Internal jdbc driver error
  SQL Error: Internal jdbc driver error
    SQL Error: Internal jdbc driver error
      SQL Error: Internal jdbc driver error
        Internal jdbc driver error
          Incomplete binding for column created
          Incomplete binding for column created

**Comment by olavloite** [comment]
The problem is that the driver is trying to convert the entire insert statement into a Cloud Spanner mutation without a roundtrip to the database. This is not possible because of the CURRENT_DATE() and CURRENT_TIMESTAMP() calls. Try this instead:

```sql
INSERT INTO providers
(provider_id, created, device_id, document_number, document_type_id, name, notification, phone, updated, last_access, deleted, token_fcm)
SELECT '000000', CURRENT_DATE(), 'not-device-id', null, null, 'provider-promocion', false, null, CURRENT_TIMESTAMP(), CURRENT_TIMESTAMP(), false, 'not-token'
;
```
This will force the driver to do a roundtrip to the server and fetch the values for CURRENT_TIMESTAMP.

**Comment by olavloite** [comment]
A more descriptive error message has been added for the next version.

**Comment by olavloite** [comment]
Fix (more descriptive error message) released in version 1.1.1.
